packages/transformers/elm/src/ElmTransformer.js

```diff
@@ -0,0 +1,125 @@
+import {Transformer} from '@parcel/plugin';
+import commandExists from 'command-exists';
+import path from 'path';
+import {minify} from 'terser';
+import ThrowableDiagnostic from '@parcel/diagnostic';
+
+export default new Transformer({
+  async transform({asset, options}) {
+    const elm = await setupElmCompiler(asset, options);
```

Comment:
Can you inline this call to `setupElmCompiler `, calling a helper function that also calls even more helper functions becomes hard to read...